app/src/test/java/org/simple/clinic/bp/entry/BloodPressureEntryLogicTest.kt

```diff
@@ -598,6 +598,7 @@ class BloodPressureEntrySheetLogicTest {
     )
```

Comment:
When we add new functionality to a screen migrated from the old legacy architecture, we should add relevant tests in the Mobius style. Can we do this for the `Update` and `ViewRenderer` changes we have introduced in this PR?

Reply:
Added tests for `ViewRenderer` and I think we already have tests for saving BP's. Can you check once?

Reply:
I meant adding `UpdateSpec` tests for verifying that the state flags are being set correctly.

Reply:
Got it